controllers/apps/components/replication/replication.go

```diff
@@ -37,6 +37,12 @@ import (
 	intctrlutil "github.com/apecloud/kubeblocks/internal/controllerutil"
 )
 
+const (
+	primaryPriority   = 1 << 2
+	secondaryPriority = 1 << 1
+	emptyPriority     = 1 << 0
```

Comment:
why do you need bits operation? I can't any bit operation associate with these? or do you just need `iota` ?